RestApi/Sepes.RestApi/Services/IAzureService.cs

```diff
@@ -8,5 +8,12 @@ public interface IAzureService
         Task<string> CreateResourceGroup(string podName);
         Task TerminateResourceGroup(string resourceGroupName);
         Task<string> CreateNetwork(string networkName, string addressSpace);
+        Task CreateSecurityGroup(string securityGroupName, string resourceGroupName);
+        Task DeleteSecurityGroup(string securityGroupName, string resourceGroupName);
+        Task ApplySecurityGroup(string resourceGroupName, string securityGroupName, string subnetName, string networkName);
+        Task RemoveSecurityGroup(string resourceGroupName, string subnetName, string networkName);
+        Task NsgAllowInboundPort(string securityGroupName, string resourceGroupName, string ruleName, int priority, string[] externalAddresses, int externalPort);
+        Task NsgAllowOutboundPort(string securityGroupName, string resourceGroupName, string ruleName, int priority, string[] internalAddresses, int internalPort);
+        Task NsgAllowPort(string securityGroupName, string resourceGroupName, string ruleName, int priority, string[] internalAddresses, int internalPort, string[] externalAddresses, int externalPort);
```

Comment:
Just a Note: Only put stuff in interface that will be used by the pod service.
This Interface looks a bit intimidating. Just a note don't need to fix now.